apps/settings/js/modules/wifi_context.js

```diff
@@ -400,10 +421,12 @@ define(function(require) {
      */
     forgetNetwork: function(network, cb) {
       cb = cb || function() {};
+      var networkKey = WifiUtils.getNetworkKey(network);
```

Comment:
please take `getNetworkKey` function from WifiUtils and avoid import it directly. It might fix test issue.